x-pack/plugins/security_solution/public/attack_discovery/attack_discovery_panel/tabs/attack_discovery_tab/index.tsx

```diff
@@ -34,21 +35,19 @@ const AttackDiscoveryTabComponent: React.FC<Props> = ({
 
   const summaryMarkdownWithReplacements = useMemo(
     () =>
-      Object.entries<string>(replacements ?? {}).reduce((acc, [key, value]) => {
-        const regex = new RegExp(key, 'g');
-
-        return acc.replace(regex, value);
```

Comment:
`acc.replaceAll(key, value)` would have been better here (and on line `47` below), though not as good as calling `replaceAnonymizedValuesWithOriginalValues`